Formula/anycable-go.rb

```diff
@@ -0,0 +1,18 @@
+class AnycableGo < Formula
+  desc "Anycable Go WebSocket Server"
+  homepage "https://github.com/anycable/anycable-go"
+  url "https://github.com/anycable/anycable-go/archive/v0.6.0.tar.gz"
+  sha256 "207734027f0a7ce4a9bfa25c30ae9576efcec1020feb15637c50f990dc18cedd"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/anycable/anycable-go/").install Dir["*"]
+    system "go", "build", "-ldflags", "-s -w -X main.version=#{version}", "-o", "#{bin}/anycable-go", "-v", "github.com/anycable/anycable-go/cmd/anycable-go"
+  end
+
+  test do
+    assert_match /#{version}/, shell_output("#{bin}/anycable-go -v")
```

Comment:
is there a chance to test more functionality? Start a server for example? You can take a look at [armor](https://github.com/Homebrew/homebrew-core/blob/16b8dd3d8c1ccf8cab179d2b967e269648d960cd/Formula/armor.rb#L29-L40) for example.

Reply:
Good idea.
Testing WebSocket server could be tricky; we're going to add http healthchecks (https://github.com/anycable/anycable-go/issues/52), we'll update the formula as soon as it released

Reply:
@palkan Is it possible to run AnyCable without redis? I don't think that it's a good idea require it explicitly in formula and rely on a default configuration. At least, starting redis automatically is an option and it's stopped right after the installation.

Reply:
Thanks Martians

Reply:
Hey @igas! Thanks for the suggestion, it has been implemented. Is there anything else  to   concider?